crates/swc_plugin_comments/src/plugin_comments_proxy.rs

```diff
@@ -0,0 +1,130 @@
+#[cfg(feature = "plugin-mode")]
+use swc_common::plugin::Serialized;
+#[cfg(feature = "plugin-mode")]
+use swc_common::{
+    comments::{Comment, Comments},
+    BytePos,
+};
+
+extern "C" {
```

Comment:
I think we need `#[cfg(target = "wasm32")]`

Reply:
It's required for testing, if the dependency on `swc_plugin` enables this code path